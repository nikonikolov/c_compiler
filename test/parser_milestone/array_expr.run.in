# 2 "arithmetic.run.in" 3 4
int x()
{}


int g, b, d, f=a;


int zz(int a, int b, int c)
{

    int arr[6][7] = { {1,2}, {3,4}, {5,6}};
    int arr[7] = { 1,2 };
    int arr[7] = { {1,2} };
}