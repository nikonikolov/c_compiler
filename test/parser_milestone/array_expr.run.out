VARIABLE : g
VARIABLE : b
VARIABLE : d
VARIABLE : f = a
FUNCTION : x
FUNCTION : zz
    PARAM : a
    PARAM : b
    PARAM : c
SCOPE
    VARIABLE : arr = {{1, 2}, {3, 4}, {5, 6}}
    VARIABLE : arr = {1, 2}
    VARIABLE : arr = {{1, 2}}
